src/types/form.ts

```diff
@@ -36,6 +36,19 @@ export enum Colors {
   Grey = 'grey',
 }
 
+export enum Rating {
```

Comment:
I'm not sure an enum is the best way to model the data here. let's say one day we decide that ratings can have increments of 0.5 - would we expand the enum with `OnePointFive = 1.5`, `TwoPointFive = 2.5` etc? doesn't seem maintainable.

furthermore, since this is a numeric enum instead of a string enum, `Object.values` actually gives you the wrong thing, as you can see in this playground demo:
![Screenshot 2021-05-24 at 12 49 15 PM](https://user-images.githubusercontent.com/29480346/119297897-763cb500-bc8e-11eb-9a45-4ba12ee7039a.png)

this is because numeric enums are two-way, meaning you can do both `Rating.One` to get `1` and `Rating[1]` to get `'One'`.

I think it might be better to just generate and export a list of strings, perhaps using lodash's `range` function.